ci/ci.yaml

```diff
@@ -156,7 +156,7 @@ jobs:
             config:
               platform: linux
               caches:
-                - path: ~/.nuget/packages
+                - path: ./cache
```

Comment:
`./` is confusing and potentially invalid in this situation, please use `pah: cache`

Reply:
Resolved in 9b8109f